pkg/version/version_test.go

```diff
@@ -0,0 +1,45 @@
+package version
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetFullVersion(t *testing.T) {
+	testCases := []struct {
+		name            string
+		expectedversion string
+	}{
+		{
+			name:            "Unknown version",
+			expectedversion: "UNKNOWN",
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+			gotversion := GetFullVersion()
+			assert.Equal(t, tc.expectedversion, gotversion)
+
+		})
+	}
+}
+func TestGetFullVersionWithRuntimeInfo(t *testing.T) {
+	testCases := []struct {
+		name            string
+		expectedversion string
+	}{
+		{
+			name:            "Unknown version",
+			expectedversion: "UNKNOWN darwin/arm64 go1.18.5",
```

Comment:
You should remove `darwin/arm64 go1.18.5`, this changes on different OS, your test will fail on linux.
You can check the output contains `UNKNOWN` using `assert.Contains`